docs/rebuild_admin.rst

```diff
@@ -127,7 +121,7 @@ chosen, you can check as follows:
  #. Log in to the *Application Server* via the console using the adminstration username
     and password.
  #. Check to see if an SSH hidden proxy service exists, using the command
-    ``sudo cat /var/lib/tor/services/ssh/hostname``. If this file exists and
+    ``sudo cat /var/lib/tor/services/sshv3/hostname``. If this file exists and
     includes an Onion URL and authorization token, your instance is set up
```

Comment:
For v3 onion services, there is no token in this file, it's just the onion address (the authorized public key is in `authorized_clients`).